src/models/room.ts

```diff
@@ -611,8 +612,8 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
      * Resolves to the version the room should be upgraded to.
      */
     public async getRecommendedVersion(): Promise<IRecommendedVersion> {
-        const capabilities = await this.client.getCapabilities();
-        let versionCap = capabilities["m.room_versions"];
+        const capabilities = this.client.getCachedCapabilities();
```

Comment:
What if `getRecommendedVersion` is called before the first capabilities response comes down?